shared/src/business/utilities/documentGenerators/docketRecord.test.ts

```diff
@@ -1,3 +1,19 @@
+/**
+ * Test suite for the `docketRecord` PDF document generator.
+ *
+ * This test verifies that the `docketRecord` function generates a printable Docket Record PDF
+ * with the provided case details, practitioners, petitioners, and docket entries.
+ * It uses `generateAndVerifyPdfDiff` to ensure the generated PDF matches the expected output.
+ *
+ * The test data includes:
+ * - Case caption and title
+ * - Docket number and suffix
+ * - Petitioner and practitioner details
+ * - A sample docket entry with formatted dates and served status
+ *
+ * @see docketRecord
+ * @see generateAndVerifyPdfDiff
+ */
```

Comment:
Don't see the harm in adding this, but it should be unnecessary if we've written our tests well

Reply:
➕